external/src/test/resources/migration/V1.0.0__POL-450_alerts_history.sql

```diff
@@ -0,0 +1,13 @@
+-- POL-450
+-- This project uses the `policies_ui_backend` database but its schema is actually managed by the policies-ui-backend project.
+-- We need that script to run standalone policies-engine tests.
+
+create table alerts_history (
+    id uuid not null,
+    tenant_id varchar(255) not null,
+    policy_id varchar(255) not null,
+    ctime bigint not null,
+    host_id varchar(255),
+    host_name varchar(255),
```

Comment:
These `varchar` max lengths probably need to be adjusted.

Reply:
Shouldn't the Id's be uuid? So something like 32 chars?

Reply:
Yes probably, but these values are currently stored as `String` and not `UUID` into Infinispan so I chose the safest path with fields capacities purposedly large (maybe a bit too much). We can easily alter the schema and move from `varchar(255)` to `uuid` (or a small `varchar`) later. I thought about "dumping" prod data after one or two weeks and check the max length of each field.